docs/platform-packet.md

```diff
@@ -18,64 +18,78 @@ is a work in progress.
 [Type 1]:https://www.packet.net/bare-metal/servers/type-1/
 [Type 2A]:https://www.packet.net/bare-metal/servers/type-2a/
 
+The `linuxjit run packet` command can mostly either be configured via
+command line options or with environment variables. see `linuxkit run
+packet --help` for the options and environment variables.
+
 ## Boot
 
-Build an image with `moby build`. The [packet.yml](https://github.com/vielmetti/linuxkit/blob/master/examples/packet.yml)
-example file provides a suitable template to start from.
+LinuxKit on
+Packet
+[boots via iPXE](https://github.com/vielmetti/linuxkit/blob/master/examples/packet.yml). It
```

Comment:
It would be better, I think, for this example not to live in https://github.com/vielmetti/linuxkit but instead in https://github.com/linuxkit/linuxkit .